api/src/main/java/com/intern/api/Greeting.java

```diff
@@ -0,0 +1,21 @@
+package com.intern.api;
+
+public class Greeting {
```

Comment:
Let's use lombok here. Add its dependency and remove constructor with getters and mark the class with `@Data` annotation`. Read about it.